messages/maintainer_access.md

```diff
@@ -0,0 +1,21 @@
+Hello {NAME},
+
+You have been nominated as {role} and the Coordinating Committee has enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}
+
+In case you are not familiar with the role, maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). In practice this means a fair amount of community interaction when reviewing PRs and related discussion.  As a general guideline, if you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project.  If necessary, you should also real free to reach out to the Coordination Committee (by @-mention on github or via coordinators@astropy.org), as the Coordination Committee has the authority to make decisions when consensus cannot be acheived.
```

Comment:
```suggestion
In case you are not familiar with the role, maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). In practice this means a fair amount of community interaction when reviewing PRs and related discussion.  As a general guideline, if you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project.  If necessary, you should also free to reach out to the Coordination Committee (by @-mention on github or via coordinators@astropy.org), as the Coordination Committee has the authority to make decisions when consensus cannot be achieved.
```